assembly/src/assembler/procedure_cache.rs

```diff
@@ -60,4 +74,22 @@ impl ProcedureCache {
             }
         }
     }
+
+    /// Inserts a re-exported [ProcedureId] to reference [ProcedureId] map into the
+    /// [ProcedureCache].
+    pub fn insert_reexported(
+        &mut self,
+        proc_id: ProcedureId,
+        ref_proc_id: ProcedureId,
+    ) -> Result<(), AssemblyError> {
```

Comment:
`alias_proc_id` seems a little confusing to me personally since it's unclear if it means the alias or the original/source procedure. Would `source_proc_id` or `original_proc_id` be better?